CHANGELOG.md

```
@@ -1,3 +1,8 @@
+### Fix
+
+- handle shit token send error []()
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)